lib/membrane/rtp/twcc_receiver.ex

```diff
@@ -150,4 +159,30 @@ defmodule Membrane.RTP.TWCCReceiver do
       }
     }
   end
+
+  # Checks if actions would be valid in the current context, namely, if they don't reference the pad that hasn't been linked yet
+  # If they are valid, they are executed immediately, otherwise they are buffered and executed as soon as the pad in question is added
+  defp send_as_actions_or_buffer(actions, ctx, state) do
```

Comment:
It's a nice, generalized solution to the problem, but I'm worried if it's an overkill for this use case. Callbacks in this element send actions on only one pad at a time, so maybe we could just check if `Map.has_key?(ctx.pads, the_pad)` and then store or send immediately all at once?

Reply:
Fine, but to minimize potential for nasty bugs in the future, the if statement will have to appear in every single callback. Otherwise it would be asking for trouble if this property ever changes

Reply:
I was thinking about passing `Map.has_key?(ctx.pads, the_pad)` as the function parameter, and then matching in the `send_as_actions_or_buffer` function header on `true`/`false`. This way it will be explicit that with this implementation we are using only one output pad at a time, and we can reduce the performance impact to the necessary minimum.

Reply:
That sounds weird to me. I think I'd prefer having the `if` statement. - that would be even more explicit
```elixir
if Map.has_key?(ctx.pads, pad), do: {{:ok, actions}, state}, else: {:ok, buffer(actions, state)}
```

Reply:
I wanted to keep the logic related to pad presence separate, but if you prefer having it in callbacks, we can go with that.